00 Intro/03 Output/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "boilerplate",
+  "name": "output",
   "version": "1.0.0",
-  "description": "Front-End Lemoncode Master, Bundle Modules, Webpack Demo 00 Boilerplate",
+  "description": "Front-End Lemoncode Master, Bundle Modules, Webpack Demo 03 Output",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server"
+    "start": "webpack-dev-server -p"
```

Comment:
Sorry I was confused because I didn't realise the comment was about using the param in the command line, not in the config file.